frontend/src/Settings/Notifications/Notifications/Notification.js

```diff
@@ -112,7 +114,14 @@ class Notification extends Component {
         }
 
         {
-          !onGrab && !onDownload && !onRename && !onHealthIssue &&
+          supportsOnDelete && onDelete &&
```

Comment:
These should be updated to ternaries (I've been slowly moving things over as I touch those files.

```
supportsOnDelete && onDelete ?
  <Label kind={kinds.SUCCESS}>
    On Delete
  </Label> :
  null
```

Reply:
Resolved in an upcoming commit.